### PR TITLE
Add new document for lists

### PR DESCRIPTION
**Pull request by GO-Jdang**
This is a document stored in a directory
    named docs

* The doc is a text file
* The file is named Lists.txt

### CODE REVIEW

File: Docs/Lists.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+ListItem 1
```
Comment (GO-Jdang) [inline Docs/Lists.txt]: Please remove the space in LineItem 1 to be LineItem1

Comment (GO-Jdang) [inline Docs/Lists.txt]: Please remove the space in LineItem 1 to be LineItem1

File: Docs/Lists.txt [hunk 2]
```diff
@@ -0,0 +1,2 @@
...
+ListItem2
```
Comment (GO-sdoki) [inline Docs/Lists.txt]: Use camelcase

File: Docs/Lists.txt (hunk 1 above)
Comment (GO-sdoki) [inline Docs/Lists.txt]: Use camelcase

**Pull request merged**